android/src/main/java/com/bitmovin/player/reactnative/BufferModule.kt

```diff
@@ -0,0 +1,73 @@
+package com.bitmovin.player.reactnative
+
+import com.bitmovin.player.api.buffer.BufferLevel
+import com.bitmovin.player.api.media.MediaType
+import com.bitmovin.player.reactnative.converter.JsonConverter
+import com.facebook.react.bridge.*
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.UIManagerModule
+
+private const val MODULE_NAME = "BufferModule"
+
+@ReactModule(name = MODULE_NAME)
+class BufferModule(private val context: ReactApplicationContext) : ReactContextBaseJavaModule(context) {
+    /**
+     * JS exported module name.
```

Comment:
I think we can remove this comment, it doesn't provide any more context on my opinion.